controllers/tenants.rb

```diff
@@ -1,9 +1,17 @@
 # index
 get '/apartments/:id/tenants' do
+  @tenants = Tenant.where(apartment_id: params[:id])
```

Comment:
Or `@apartment = Apartment.find(params[:id])`

and then

`@apartment.tenants`